lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -275,9 +280,12 @@ sub find_jobs_with_expired_logs ($self, $preserved_important_jobs_out = undef) {
 
 # helper function for cleanup task
 sub limit_results_and_logs ($self, $preserved_important_jobs_out = undef) {
-    my $expired_jobs = $self->find_jobs_with_expired_results;
+    my $expired_jobs = $self->find_expired_jobs;
     $_->delete for @$expired_jobs;
 
+    my $expired_results_jobs = $self->find_jobs_with_expired_results;
+    $_->delete_results for @$expired_results_jobs;
```

Comment:
We are currently also calling `delete_results` as part of the `ensure_results_below_threshold` task:
https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Task/Job/Limit.pm#L237
https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Task/Job/Limit.pm#L253

Is it intentional that we have two places for that now?